src/include/execution/vm/bytecodes.h

```diff
@@ -172,6 +172,15 @@ namespace terrier::execution::vm {
                                                                                                                       \
   /* SQL type comparisons */                                                                                          \
   F(ForceBoolTruth, OperandType::Local, OperandType::Local)                                                           \
+  F(IsNull, OperandType::Local, OperandType::Local)                                                                   \
+  F(IsNotNull, OperandType::Local, OperandType::Local)                                                                \
+  F(InitNullBool, OperandType::Local)                                                                                 \
+  F(InitNullInt, OperandType::Local)                                                                                  \
+  F(InitNullReal, OperandType::Local)                                                                                 \
+  F(InitNullDecimal, OperandType::Local)                                                                              \
```

Comment:
For consistency, mimic the `Init*` bytecodes by adding the `Val` suffix for SQL values?